.gitignore

```diff
@@ -1,6 +1,8 @@
 # Dev
 composer.lock
 vendor/
+.idea
```

Comment:
This should not be included into project `.gitignore`. Toolings ignores should be declared on your local machine globally

https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Reply:
Ok, I'll fix it

Reply:
It is removed.